app/controllers/recipe_controller.rb

```diff
@@ -7,8 +7,9 @@ class RecipeController < ApplicationController
   respond_to :json
   #Show all recipes
   def index
-    @recipes = Recipe.all
-    respond_with( @recipes )
+    #return all recipes sorted by date of publication.
+    @recipes = Recipe.order(created_at: :desc).all.paginate(:page => params[:page], :per_page => 10)
```

Comment:
No need to `.all`  the `order()` will return a collection 
and plz remove  ruby 1.8 syntax
```ruby
@recipes = Recipe.order(created_at: :desc).paginate( page:         params[:page],                         
                                                                                       per_page: 10 )
```